Automattic/vip-cli
# Removing custom wp-config-defaults

## Description

We did a workaround a while ago to support some customization over the `wp-config-defaults.php` file on the dev environment. The 

This PR depends on this other one being merged: https://github.com/Automattic/vip-container-images/pull/5

## Steps to Test

1. Check out PR and build the CLI.
2. Check that a `wp-config-defaults.php` file exists in `vip-local/config` and that you can modify it.
3. Run the application normally.
